bin/faucet/src/config.rs

```diff
@@ -39,8 +41,8 @@ pub struct FaucetConfig {
 impl Display for FaucetConfig {
     fn fmt(&self, f: &mut Formatter<'_>) -> std::fmt::Result {
         f.write_fmt(format_args!(
-            "{{ endpoint: \"{}\", node_url: \"{}\", timeout_ms: \"{}\", asset_amount_options: {:?}, faucet_account_path: \"{}\", remote_tx_prover_url: \"{:?}\" }}",
-            self.endpoint, self.node_url, self.timeout_ms, self.asset_amount_options, self.faucet_account_path.display(), self.remote_tx_prover_url
+            "{{ endpoint: \"{}\", node_url: \"{}\", timeout_ms: \"{}\", asset_amount_options: {:?}, faucet_account_path: \"{}\", pow_salt: \"{}\", remote_tx_prover_url: \"{:?}\" }}",
+            self.endpoint, self.node_url, self.timeout_ms, self.asset_amount_options, self.faucet_account_path.display(), self.pow_salt, self.remote_tx_prover_url
```

Comment:
Does this have to be manually input? It would be nice if `miden-faucet init` already generated a random secret.

Reply:
Sounds good 👌🏼